webapp/src/modules/estates/actions.js

```diff
@@ -138,7 +150,11 @@ export function deleteEstateSuccess(txHash, estate) {
   return {
     type: DELETE_ESTATE_SUCCESS,
     ...buildTransactionAction(txHash, {
-      estate,
+      estate: {
+        id: estate.id,
+        data: { name: estate.data.name, parcels: estate.data.parcels }
```

Comment:
Maybe we could have a utils fn to map the parcels leaving only the necessary props `(x,y)` ?

Reply:
Parcels have an object `{ x, y }`  here